### PR TITLE
[#9117] Delete course/session: remove the term 'soft-delete' from UI

### LINKED ISSUE #9117: Instructor: delete course/session: remove the term 'soft-delete' from UI
v6.11.0

I noticed the term 'soft-delete' in the UI while running UI tests. I don't think that's a term meaningful to users.

Suggested: replace with another term such as 'deleted' or 'in the recycle bin'

### PR DESCRIPTION
**Pull request by kumarav1nash**
Fixes #9117

**PR Checklist**
 -Followed all the checklist for PR
**Outline of Solution**

Search for 'soft-delete'  in the web directory found the paragraph that needed to be updated.
replaced it with 'delete' term.

### CODE REVIEW

**Comment by moziliar** [comment]
As a general guide, UI changes PR usually comes with a before and after screenshot of the UI affected. 
The change LGTM.

**Comment by kumarav1nash** [comment]
@moziliar, in that case, I might need a login credential to log in as a teacher in my local environment.
how am I suppose to get one?

**Comment by moziliar** [comment]
@kumarav1nash you can refer to this [link]( https://github.com/TEAMMATES/teammates/blob/master/docs/development.md#logging-in-to-a-teammates-instance) see how you could masquerade as an admin, adding instructor, and operate as an instructor!

Do feel free to ask any questions.

**Comment by kumarav1nash** [comment]
@moziliar  thanks for the help, I've tried masquerading as an admin but unfortunately I'm not able to log in

**Comment by moziliar** [comment]
@kumarav1nash Hi, can you share with me what you have tried? Have you attempted all the steps in the documentation?
The way to create "credentials" for local development is simply creating users while masquerading as the admin.

**Comment by kumarav1nash** [comment]
@moziliar so i went to this address `http://localhost:8080/web/admin/home` but I'm getting login screen with email input field and login button.there I used default mail id `test@example.com` and then i got an alert message you are not authorized to login. I've also tried to masquerade after this step as mentioned in docs but no success

**Comment by hhdqirui** [comment]
@kumarav1nash Hi. You can try to go to the file `build.properties` and update the `app.admin` to something like `app.admin=test@example.com`. This may help.

**Comment by kumarav1nash** [comment]
@hhdqirui Hi, I've tried this but still getting the same issue

**Comment by kumarav1nash** [comment]
Hi, it's working now,  earlier I got confused since it was asking for email, but when I entered the default value as `app_admin`(mentioned in build.properties)  in the email input field I got validated successfully.

**Comment by t-cheepeng** [comment]
@kumarav1nash 
Once you are able to, please do post a before/after screenshot of the changes. Thanks

**Comment by moziliar** [comment]
^ picking up from @hhdqirui's advice, you can also just use `app_admin` predefined:)

**Comment by kumarav1nash** [comment]
Hi, Here's the screenshot of before and after changes, Look at the Note section, there you will find the change.
Thanks.

Before change

Before Change image

After change

After Change image

**Review by jianhandev — APPROVED** [review]
LGTM

**Comment by kumarav1nash** [comment]
Hi, I'm kind of new to open source contributions, what I'm supposed to do next? This PR seems successfully merged, should I close this PR?

**Comment by moziliar** [comment]
Hi, you will just have to wait for it to be merged during our release. No further action is required from you!

**Comment by kumarav1nash** [comment]
okay got it, Thanks

File: src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
   <div class="row">
     <div class="col-12 text-muted">
       <p>
-        Note: The table above doesn't contain sessions from archived courses and soft-deleted courses.
+        Note: The table above doesn't contain sessions from archived courses and deleted courses.
```
Comment (wkurniawan07) [inline src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html:47]: This doesn't blend well with the sentence right after it: it says "course in recycle bin" vs here where it says "deleted courses", and those two could (and actually do) mean different things.

**Review by wkurniawan07 — CHANGES_REQUESTED** [review]

File: src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html (hunk 1 above)
Comment (kumarav1nash) [reply src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html:47]: Hi, what do you suggest?

Comment (wkurniawan07) [reply src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html:47]: Following the succeeding sentence: "courses in recycle bin"

**Review by wkurniawan07 — APPROVED** [review]
LGTM, thanks for contributing!

**Comment by kumarav1nash** [comment]
Thanks, would love to contribute to this project again

**Pull request merged**